src/link/C/zig.h

```diff
@@ -500,3 +500,21 @@ zig_shl_sat_s(isize, intptr_t, ((sizeof(intptr_t)) * CHAR_BIT - 1))
 zig_shl_sat_s(short,    short, ((sizeof(short   )) * CHAR_BIT - 1))
 zig_shl_sat_s(int,        int, ((sizeof(int     )) * CHAR_BIT - 1))
 zig_shl_sat_s(long,      long, ((sizeof(long    )) * CHAR_BIT - 1))
+
+#define bitsizeof(T) (CHAR_BIT * sizeof(T))
```

Comment:
don't pollute the global namespace:

```suggestion
#define zig_bitsizeof(T) (CHAR_BIT * sizeof(T))
```

Reply:
Done.